File: tasks/CodeForces/R1/TaskA.cpp
#include "global.hpp"
#include "Scanner.hpp"

class TaskA {
public:
    void solve(std::istream& inStream, std::ostream& outStream) {
        Scanner in(inStream);
        Writer out(outStream);

        int n, m, a;
        in >> n >> m >> a;
        int64 need1 = n / a;
        if (n % a != 0) {
            need1++;
        }
        int64 need2 = m / a;
        if (m % a != 0) {
            need2++;
        }
        int64 result = need1 * need2;
        out << result;
    }
};

File: java/src/main/TaskA.java
package main;

import java.util.Scanner;
import java.io.PrintWriter;

public class TaskA {
    public void solve(int testNumber, Scanner in, PrintWriter out) {
        int n = in.nextInt();
        int result = n / 2;
        out.println(result);
        for (int i = 0; i < result - 1; i++) {
            out.print(2 + " ");
        }
        out.print(n % 2 == 0 ? 2 : 3);
    }
}
